onnxruntime/core/graph/featurizers_ops/featurizers_defs.cc

```diff
@@ -1373,6 +1374,61 @@ void RegisterNormalizeFeaturizerVer1() {
           });
 }
 
+void RegisterShortGrainDropperFeaturizerVer1() {
+  // static const char* doc = R"DOC(
+  //     Drop rows of TimeSeriesDataFrame with short grains
+
+  //     C++-style pseudo signature:
+  //       bool execute(std::vector<std::string> const &value);
+
+  //     Examples:
+  //       todo:
+  // )DOC";
+  MS_FEATURIZERS_OPERATOR_SCHEMA(ShortGrainDropperTransformer)
+      .SinceVersion(1)
+      .SetDomain(kMSFeaturizersDomain)
+      .Input(
+          0,
+          "State",
+          "State generated during training that is used for prediction",
+          "T0")
+      .Input(
+          1,
+          "Input",
+          "No information is available",
```

Comment:
>No information is available [](start = 11, length = 27)

Need information on the shape of the input so the type and shape inference can be verified.